tools/cldr-apps/js/src/esm/cldrGui.js

```diff
@@ -67,7 +67,13 @@ async function ensureSession() {
     // We want to delete this parameter. It either is used and no longer needed,
     // or it is invalid.
     params.delete("s");
-    window.location.search = params.toString();
+    let newQuery = '';
+    if (params.toString()) {
```

Comment:
setting window.location.search is problematic for sure -- it causes ensureSession to execute twice, throwing away the good session from the "info" request and replacing it with a bad session (no user) from the "login" request